segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorage.java

```diff
@@ -597,6 +614,7 @@ private void collectGarbage(Collection<String> chunksTodelete) {
                 // Validate preconditions.
                 checkSegmentExists(streamSegmentName, segmentMetadata);
                 checkOwnership(streamSegmentName, segmentMetadata);
+                checkChunksExist(txn, segmentMetadata);
```

Comment:
Do all of these calls result in LTS or Table Segment calls? You've added quite a bit of now.

Reply:
I'll probably change it to int debugLevel

Reply:
on second thought, I removed these checks - not really needed anymore.